src/main/java/data/JsonConverter.java

```diff
@@ -136,7 +136,7 @@ public String getConfigMap() throws IOException {
   }
 
   /** Return list of vendor IDs that exist in the filesystem.*/
-  private ArrayList<String> getVendorIDs() {
+  public ArrayList<String> getVendorIDs() {
     File root = new File(basePath);
     return new ArrayList<String>(Arrays.asList(Objects.requireNonNull(root.list())));
   }
```

Comment:
could you add a test for this method now that it's public?

Reply:
Sure.